docs/docs/38-authoring/21-services.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Services
+---
+
+## Consuming services
+
+When you are authoring the Acornfile for your application you can define cloud servics that will be provisioned for your application. The services will be deployed alongside your application at deploy/run time. To learn how to create your own service Acorns see [services](/100-reference/10-services.md) in the reference section.
+
+### Wiring services into your Acorn app
+
+Service attritibutes are accessed through the `@{}` syntax in the Acornfile. Here is a simple example of accessing the `address` attribute of a service named `db`. For complete service syntax see the [services](/100-reference/03-acornfile.md#services-consuming) section in the Acornfile reference.
+
+```acorn
+
+```acorn
+// This service exposes an address and a secret
+services: db: {
+    image: "ghcr.io/acorn-io/aws/rds-aurora-cluster:latest"
+}
+
+containers: app: {
+    image: "my-app:latest"
+    env: {
+        MY_SERVICE_ADDRESS: "@{service.db.address}"
+        DB_USER: "@{service.db.secrets.admin.username}"
+        DB_PASS: "@{service.db.secrets.admin.password}"
```

Comment:
DB_USER: "@{secrets.db.admin.username}"
DB_PASS: "@{secrets.db.admin.password}"

Reply:
For simplicity, I chose the service.[name].secrets.X.X format. I wanted to avoid confusing users by showing two top-level keys to access data from the service. Both styles work though.